packages/sitecore-jss/src/editing/index.ts

```diff
@@ -6,8 +6,10 @@ export {
   resetEditorChromes,
   handleEditorAnchors,
   Metadata,
+  getJssHorizonClientData,
   EDITING_ALLOWED_ORIGINS,
   QUERY_PARAM_EDITING_SECRET,
+  PAGES_EDITING_MARKER,
```

Comment:
Probably we don't need to expose this variable, since it's an internal thing

Reply:
we use it in sitecore-jss-react (and will in sitecore-jss-angular, later). but it isn't exported further.